### ISSUE TITLE
Lava cauldrons do not emit light; looks very dark in comparison to actual lava

### ISSUE THREAD

**Issue by EeveeEuphoria** [sixthsurge/photon#74]
### What happened?

Basically what's in the title, dunno how else to explain this. I set my settings to the defaults found in "Ultra", most of everything else looks fine except for this. I'm just assuming there's missing data on this.

![2023-11-06_20 12 43](https://github.com/sixthsurge/photon/assets/8446547/2a57b4ff-d256-4236-adaa-2aeef7f6cd4e)

![2023-11-06_20 14 30](https://github.com/sixthsurge/photon/assets/8446547/1773f85f-85b5-4d32-9dab-a6905dce5d18)

### Minecraft Version

1.20.1

### Which shader mod are you using?

Iris

### Shader mod version

1.6.10

### Shader settings

```conf
#Mon Nov 06 20:11:12 EST 2023
COLORED_LIGHTS=true
shadowMapResolution=4096
HANDHELD_LIGHTING=true
BLOCK_ENTITY_SHADOWS=true
CLOUD_SHADOWS=true
INFO=3
WATER_CAUSTICS=true
AIR_FOG_COLORED_LIGHT_SHAFTS=true
```

**Comment by Patomosm** [comment]
Yes, lava cauldron in the center of the image does not emit light.
![Base Profile Screenshot 2023 11 08 - 17 17 42 74](https://github.com/sixthsurge/photon/assets/133518090/101789e6-2ff9-4e86-8b12-f467362d35a6)

**Comment by WhyFenceCode** [comment]
#106 
Awaiting Approval For Commit to Fix This, Hopefully within 1-2 weeks

**Comment by sixthsurge** [comment]
fixed by #106
